docs/sources/mimir/release-notes/v2.14.md

```diff
@@ -0,0 +1,139 @@
+
+Following featues of store-gateway are changed: `-blocks-storage.bucket-store.max-concurrent-queue-timeout` is set to 5 seconds;
+`-blocks-storage.bucket-store.index-header.lazy-loading-concurrency-queue-timeout` is set to 5 seconds;
+`-blocks-storage.bucket-store.max-concurrent` is set to 200;
+
+Number of previously deprecated configuration options were removed in the release:
+
+- the `-ingester.return-only-grpc-errors` option in ingester
+- the `-ingester.client.circuit-breaker.*` options in ingester
+- the `-ingester.limit-inflight-requests-using-grpc-method-limiter` option in ingester
+- the `-ingester.client.report-grpc-codes-in-instrumentation-label-enabled` option in distributor and ruler
+- the `-distributor.limit-inflight-requests-using-grpc-method-limiter` option in distributor
+- the `-distributor.enable-otlp-metadata-storage` option in distributor
+- the `-ruler.drain-notification-queue-on-shutdown` option in ruler
+- the `-querier.max-query-into-future` option in querier
+- the `-querier.prefer-streaming-chunks-from-store-gateways` option in querier and store-gateway
+- the `-query-scheduler.use-multi-algorithm-query-queue` option in querier-scheduler
+- the YAML configuration `frontend.align_queries_with_step` in query-frontend
+
+## Experimental features
+
+Grafana Mimir 2.14 includes new features that are considered experimental and disabled by default.
+Use them with caution and report any issues you encounter:
+
+A new experimental Kafka-based ingest storage architecture is added, that decouples the write and read path through a Kafka-compatible backend.
+When enabled, distributors write incoming write requests to the Kafka-compatible backend, and the ingesters asynchronously replay ingested data from Kafka.
+
+New related configuration options that enable and configure the new architecture in Mimir are: `-ingest-storage.enabled`, `-ingest-storage.kafka.*`,
+`-ingest-storage.ingestion-partition-tenant-shard-size`, `-ingest-storage.read-consistency`, `-ingest-storage.migration.distributor-send-to-ingesters-enabled`,
+`-ingester.partition-ring.*`.
+
+Querier has an experimental streaming PromQL engine. The configuration option `-querier.query-engine=mimir` enables the engine.
+
+Ingester added experimental `-ingester.ignore-ooo-exemplars` configuration. When set, the out of order exemplars are no longer reported
+to the remote write client.
+
+Querier now supports the experimental `limitk()` and `limit_ratio()` PromQL functions. The feature is disabled by default,
+but can be enabled with the `-querier.promql-experimental-functions-enabled=true` setting in the query-frontend and querier.
+
+Querier supports for the experimental `X-Mimir-Chunk-Info-Logger` header, that triggers logging information about TSDB chunks loaded from ingesters and store-gateways in the querier.
+The header should contain the comma separated list of labels, for which their value will be included in the logs.
```

Comment:
I'm not sure this is worth mentioning, this is a feature intended to aid with debugging incorrect query results.